server/src/main/java/org/example/MyGRPCServer.java

```diff
@@ -0,0 +1,21 @@
+package org.example;
+
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import org.example.service.MyServiceImpl;
+
+import java.io.IOException;
+
+public class MyGRPCServer {
+    public static void main(String[] args) throws IOException, InterruptedException {
+
+        Server server = ServerBuilder
+                .forPort(8080)
```

Comment:
Optional: create properties file, place configuration for port over there and read that file in the main method.